apps/snitch_core/config/dev.exs

```diff
@@ -6,7 +6,7 @@ config :snitch_core, Snitch.Repo,
   username: "postgres",
   password: "postgres",
   database: "snitch_dev",
-  hostname: "localhost",
+  hostname: "db",
```

Comment:
This should be picked up from system env otherwise will fail for manual setup

Reply:
@ashish173 Yes. I and @Jeshan7 has discussed this and he is currently working on the changes.